docs/templates/table.md.go.tpl

```diff
@@ -40,5 +40,5 @@ The following tables depend on {{.Name}}:
 | Name          | Type          |
 | ------------- | ------------- |
 {{- range $column := $.Columns }}
-|{{$column.Name}}{{if $column.PrimaryKey}} (PK){{end}}{{if $column.IncrementalKey}} (Incremental Key){{end}}|{{$column.Type}}|
+|{{$column.Name}}{{if $column.PrimaryKey}} (PK){{end}}{{if $column.IncrementalKey}} (Incremental Key){{end}}|`{{$column.Type}}`|
```

Comment:
don't we need to account for this in https://github.com/cloudquery/cloudquery/tree/main/scripts/table_diff
cc: @yevgenypats @hermanschaaf

Reply:
This was an old change that we lost somehow, we do account for it in that script 👍